packages/react-instantsearch-dom/src/components/PoweredBy.js

```diff
@@ -7,70 +7,73 @@ import { createClassNames } from '../core/utils';
 const cx = createClassNames('PoweredBy');
 
 /* eslint-disable max-len */
-const AlgoliaLogo = () => (
+const AlgoliaLogo = ({ theme }) => (
   <svg
+    width="168"
     xmlns="http://www.w3.org/2000/svg"
-    baseProfile="basic"
-    viewBox="0 0 1366 362"
-    width="100"
-    height="27"
```

Comment:
the width and height should probably stay the same as before, to avoid breaking change?

Reply:
I think it is the same size as before but the SVG isn't, I'll try to see if I can set the same width/height

Reply:
I think the height matches, but the new width is a tiny bit more. Not sure whether it's worth worrying about